spec/models/blood_pressures_per_facility_per_day_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe BloodPressuresPerFacilityPerDay, type: :model do
+  describe 'Associations' do
+    it { should belong_to(:facility) }
+  end
+
+  describe 'Materialized view query' do
+    let!(:facility_with_bp) { create(:facility) }
+    let!(:patient) { create(:patient, registration_facility: facility_with_bp) }
+    let!(:patient_2) { create(:patient, registration_facility: facility_with_bp) }
+    let!(:facility_without_bp) { create(:facility) }
+
+    let!(:days) do
+      [1, 2].map { |n| n.days.ago }
+    end
+
+    let!(:blood_pressures) do
+      days.map do |day|
+          create_list(:blood_pressure, 2, facility: facility_with_bp, recorded_at: day, patient: patient)
+      end + [create(:blood_pressure, facility: facility_with_bp, recorded_at: days.first, patient: patient_2)]
+    end
+
+    before do
+      described_class.refresh
+    end
+
+    it 'should return a row per facility per day' do
```

Comment:
Can we remove the "should"s from the spec descriptions? It's generally better (and fewer keystrokes) to have the descriptions read something like `it 'returns a row per facility per day'`, etc.